src/hooks/useLoadDocuments.ts

```diff
@@ -35,4 +36,13 @@ const useLoadDocuments = (trainingSetId: number): ReturnType<DocumentType[]> =>
   return transformArticle(documents)
 }
 
+export const useLoadDocumentsRandomOrder = (trainingSetId: number): ReturnType<DocumentType[]> => {
+  const documents: ReturnType<DocumentType[]> = useLoadDocuments(trainingSetId)
+  if (!documents.data) {
```

Comment:
you can simplify this to `return documents.data ? { ...documents, data: shuffleArray(documents.data) } : documents`